tests/unit/test_rendering.py

```diff
@@ -113,5 +113,11 @@ def test_form_rendering(self):
         self.assertTrue('Mac OS X 10.13' in rv.data)
         self.assertTrue('http://example.com/' in rv.data)
 
+    def test_wellknown_subpath(self):
+        """Test that the /.wellknown/subpath route gets 404."""
+        rv = self.app.get('/.well-known/{subpath}')
```

Comment:
here the code is testing a `{subpath}` variable, which is not we are trying to do. The `{subpath}` in the flask route is a keyword we are using later on in the template. See [variable rules](http://flask.pocoo.org/docs/1.0/quickstart/#variable-rules)

We are trying to test if any string after `/.well-known/` will trigger a 404. For example `/.well-known/wonderland` or `/.well-known/jupiter` really anything should trigger a 404.

So you can name any tests routes, the name doesn't matter.